bin.js

```diff
@@ -43,25 +43,25 @@ if (argv[0] == 'start') {
   // import ssbServer and start the server
 
   var createSsbServer = require('./')
-    .use(require('./plugins/onion'))
-    .use(require('./plugins/unix-socket'))
-    .use(require('./plugins/no-auth'))
-    .use(require('./plugins/plugins'))
-    .use(require('./plugins/master'))
+    .use(require('ssb-onion'))
+    .use(require('ssb-unix-socket'))
+    .use(require('ssb-no-auth'))
+    .use(require('ssb-server-plugins'))
```

Comment:
note there is already a ssb-plugins module underway

https://github.com/ssbc/ssb-server/pull/653

Reply:
Yep, I noticed that when I tried to make an `ssb-plugins` repo and saw that it already existed. I'm using the name "ssb-server-plugins" to denote "this is literally the code from ssb-server" but I'm excited to have it deprecated once you feel ssb-plugins is ready for wider usage. :+1:

Reply:
I got it working on the plane, so i will merge these both in a sec